test/src/test/java/org/corfudb/runtime/clients/LogUnitClientTest.java

```diff
@@ -176,9 +177,11 @@ public void CorruptedDataReadThrowsException() throws Exception {
         String logDir = (String) serverContext.getServerConfig().get("--log-path");
         String logFilePath = logDir + File.separator + "log/0.log";
         RandomAccessFile file = new RandomAccessFile(logFilePath, "rw");
-        final long FILE_HEADER = 64;
+        ByteBuffer headerBuf = ByteBuffer.allocate(Integer.BYTES);
+        file.getChannel().read(headerBuf);
+        headerBuf.flip();
         final int CORRUPT_BYTES = 0xFFFF;
-        file.seek(FILE_HEADER + 2); // File header + delimiter
+        file.seek(headerBuf.getInt() + 20); // File header + delimiter
```

Comment:
seems like this magic number is causing the build to fail. Is there a way to get this num from a protobuf?